### DISCUSSION TITLE
updating budibase from within budibase doesn't work

### DISCUSSION THREAD

**Discussion by sghriyadh** [Budibase/budibase#11604] [Help] [answered]
Hi

i downloaded docker single image and have been running same image for last three months.

inside Budibase admin panel, there is a button to update version. It shows me that there is  new version available but when i click it it instantly says budibase updated but infact it didn't do anything. 

Can anyone tell me if this button to update actually updates budibase if budibase was running inside docker ?

**Comment by jacobmaussocc** [comment, marked as answer]
I had the same issue, I ended up just re-pulling my docker image. Not sure why it doesn't work but updating through docker is probably your solution.
